test/e2e_tests/dir_test.go

```diff
@@ -501,6 +514,12 @@ func TestDirTestSuite(t *testing.T) {
 	if adlsPtr == "true" || adlsPtr == "True" {
 		fmt.Println("ADLS Testing...")
 		dirTest.adlsTest = true
+	} else if dataValidationFileSharePtr == "true" || dataValidationFileSharePtr == "True" {
```

Comment:
I think this should be fileSharePtr instead of dataValidationFileSharePtr in the condition.